docs/en/verifying-signatures.wml

```diff
@@ -9,314 +9,641 @@
+   </span>
+   </p>
+
+<!-- Windows -->
+  <div>
+    <input id="ac-4-1" name="accordion-4" type="radio" checked />
+   <article class="ac-os">
+    <!--<pre id="ttb-key">
+    > gpg.exe --keyserver pool.sks-keyservers.net --recv-keys 0x4E2C6E8793298290
+    </pre>-->
+    <h3>Verififcation with a checksum on Windows</h3>
+    <h4>Verify the signature of the checksum file</h4>
+    <p>
+    Verify the sha256sums-unsigned-build.txt file by executing this command:
+    </p>
+    <pre>
+    > gpg.exe --verify sha256sums-unsigned-build.txt.asc sha256sums-unsigned-build.txt
+    </pre>
 
-    <a id="BuildVerification"></a>
-    <h3><a class="anchor" href="#BuildVerification">
-    Verifying sha256sums (advanced)</a></h3>
-    <hr>
-    <p>Build reproducibility is a <a
-       href="https://blog.torproject.org/blog/deterministic-builds-part-one-cyberwar-and-global-compromise">security
-       property</a> of Tor Browser 3.0 and later. Anyone can build
-       Tor Browser on their own machine and produce a binary that is
-       bit-for-bit identical to the binary we offer on the download page.
-       Fortunately, it is not necessary for everyone to build Tor Browser
-       locally to get this security. Verifying and comparing the signed list
-       of <a href="https://en.wikipedia.org/wiki/Cryptographic_hash">hashes</a>
-       will confirm that multiple people have built Tor Browsers
-       identical to the download.</p>
-
-      <p>The steps below walk through this process:</p>
+    <p>
+    You should see a message like
+    <pre>"Good signature from &lt;DEVELOPER NAME&gt;"</pre>.
+    If you don't, there is a problem. Try these steps again.
+    </p>
+
+    <h4>Verify the file integrity of Tor Browser</h4>
+    <p>
+    If you want to verify a Windows Tor Browser package you need to first
+    strip off the authenticode signature of it.<br/>
+    Tools that can be used for this purpose are
+    <a href="http://osslsigncode.sourceforge.net">osslsigncode</a> and
+    <a href="http://forum.xda-developers.com/showthread.php?t=416175">delcert.exe</a>.
+    Assuming you have built e.g. <tt>osslsigncode</tt> on a Linux computer you can enter
+    </p>
+    <pre>
+    > C:\path\to\osslsigncode remove-signature &#92;
+        where\you\saved\\&lt;TOR BROWSER FILE NAME&gt;.exe &lt;TOR BROWSER FILE NAME&gt;.exe
+    </pre>
 
+    <p>Now use the sha256sum of the Tor Browser package with the
+    <a href="http://md5deep.sourceforge.net/">hashdeep utility</a> and run
+    </p>
+    <pre>
+    C:\location\of\hashdeep -c sha256sum &lt;TOR BROWSER FILE NAME&gt;.exe
+    </pre>
+   </article>
+  </div>
+<!-- Mac OS --><!--
+ <div>
+   <article class="ac-os">
+     This process does not work on macOS yet due to Apple's codesigning requirement.
+   </article>
+ </div>
+-->
+
+<!-- Linux -->
+  <div>
+   <input id="ac-4-3" name="accordion-4" type="radio" />
+   <article class="ac-os ac-4-3">
+    <!--<pre id="ttb-key">
```

Comment:
You should feel free to delete obsolete instructions.
We can always get them back out of git.